tests/phpunit/includes/PluginStatusTrait.php

```diff
@@ -1,42 +0,0 @@
-<?php
-/**
- * PluginStatusTrait
- *
- * @package   Google\Site_Kit\Tests
- * @copyright 2025 Google LLC
- * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
- * @link      https://sitekit.withgoogle.com
- */
-
-namespace Google\Site_Kit\Tests;
-
-trait PluginStatusTrait {
-
-	protected $active_plugins = array();
-
-	protected $active_plugins_callback;
-
-	public function activate_plugin( $plugin_path ) {
-		if ( ! in_array( $plugin_path, $this->active_plugins, true ) ) {
-			$this->active_plugins[] = $plugin_path;
-		}
-
-		if ( null === $this->active_plugins_callback ) {
-			$this->active_plugins_callback = array( $this, 'filter_active_plugins' );
-			add_filter( 'pre_option_active_plugins', $this->active_plugins_callback );
-		}
```

Comment:
This trait has been removed primarily because it's no longer needed, but also because it should be done a little differently to avoid state leaking between tests as it would by using a property like this. A safer approach would be to use a pattern similar to `\Google\Site_Kit\Tests\TestCase::enable_feature`.